component/modal/modal.css

```diff
@@ -0,0 +1,63 @@
+.mantra-modal {
+    display: flex;
+    flex-direction: column;
+    width: var(--modal-width);
+    border-radius: 5px;
+    box-shadow: -1px 1px 8px 1px rgb(0 0 0 / 75%);
```

Comment:
You can use this value in the common variables named as modal-box-shadow and use it here.

Reply:
Done